src/drivers/imx/sdma.c

```diff
@@ -95,10 +97,13 @@ static int sdma_run_c0(struct dma *dma, uint8_t cmd, uint32_t buf_addr, uint16_t
 	/* 1 is BIT(0) for channel 0, the bit will be cleared as the
 	 * channel finishes
 	 */
-	ret = poll_for_register_delay(dma_base(dma) + SDMA_STOP_STAT, 1, 0, 500);
+	ret = poll_for_register_delay(dma_base(dma) + SDMA_STOP_STAT, 1, 0, 400000);
```

Comment:
(I have to figure out a good polling interval that isn't huge but is enough for the actual transfers to work; perhaps one milisecond suffices but this needs testing)